playbooks/katello_pulp3.yml

```diff
@@ -0,0 +1,7 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - katello_pulp3
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
```

Comment:
Since you're repeating this, should this be part of the role and a variable?

Reply:
doesn't seem repeated anymore